assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetGA4/UserCountGraph.js

```diff
@@ -99,7 +102,7 @@ export default function UserCountGraph( props ) {
 		select( MODULES_ANALYTICS_4 ).getPropertyID()
```

Comment:
We also need to avoid selecting the `propertyID` as this will invoke `getSettings` which will error in a view-only context. Since the property ID is only used in the selector below for `getProperty` we can move that inside and make them both conditional.

Reply:
Updated, thank you!